src/HTMLRenderer/form.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * form.cc
+ *
+ * Handling Forms
+ *
+ * by Simon Chenard
+ * 2014.07.25
+ */
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "HTMLRenderer.h"
+#include "util/namespace.h"
+#include "util/misc.h"
+
+namespace pdf2htmlEX {
+   
+using std::ofstream;
+using std::cerr;
+
+void HTMLRenderer::process_form(ofstream & out)
+{
+    FormPageWidgets * widgets = cur_catalog->getPage(pageNum)->getFormWidgets();
+    int num = widgets->getNumWidgets();
+
+    for(int i = 0; i < num; i++)
+    {
+        FormWidget * w = widgets->getWidget(i);
+        double x1, y1, x2, y2, width, font_size;
```

Comment:
I fixed the rest, but this one I do not understand. Should I have declared widgets  in the header file?